catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/authentication/kerberos/KerberosConfig.java

```diff
@@ -95,12 +95,18 @@ public int getFetchTimeoutSec() {
       new ImmutableMap.Builder<String, PropertyEntry<?>>()
           .put(
               KEY_TAB_URI_KEY,
-              PropertyEntry.stringImmutablePropertyEntry(
-                  KEY_TAB_URI_KEY, "The uri of key tab for the catalog", false, null, false, false))
+              PropertyEntry.stringPropertyEntry(
+                  KEY_TAB_URI_KEY,
+                  "The uri of key tab for the catalog",
+                  false,
```

Comment:
can you please cheange to this style `false /* xxx */`, better for reviewers to understand which to which.

Reply:
done

Reply:
I found that you just changed some of the parameters, not all. It would be better to change all the boolean values. Also, don't need to explain, just `/* <parameter_name> */` is enough.